functions/Invoke-DbaDbDataGenerator.ps1

```diff
@@ -373,23 +383,26 @@ function Invoke-DbaDbDataGenerator {
                                     }
 
                                     if (($columnobject.MinValue -or $columnobject.MaxValue) -and ($columnobject.ColumnType -match 'date')) {
-                                        $nowmin = $columnobject.MinValue
-                                        $nowmax = $columnobject.MaxValue
-                                        if (-not $nowmin) {
-                                            $nowmin = (Get-Date -Date $nowmax).AddDays(-365)
+                                        if (-not $columnobject.MinValue) {
+                                            $columnobject.MinValue = (Get-Date -Date $columnobject.MaxValue).AddDays(-365)
                                         }
-                                        if (-not $nowmax) {
-                                            $nowmax = (Get-Date -Date $nowmin).AddDays(365)
+                                        if (-not $columnobject.MaxValue) {
+                                            $columnobject.MaxValue = (Get-Date -Date $columnobject.MinValue).AddDays(365)
                                         }
                                     }
 
                                     try {
+                                        # TODO: I think this is not needed, as $columnValue is set in both branches of the following if.
```

Comment:
I was very cautious to not have a value in that parameter because it was used so often. I think this part can be removed.